src/container/srv_container.c

```diff
@@ -2420,21 +2422,23 @@ ds_cont_op_handler(crt_rpc_t *rpc)
 	daos_prop_t	       *prop = NULL;
 	struct cont_svc	       *svc;
 	int			rc;
+	uuid_t			pool_uuid;
```

Comment:
Hmm, why do we need to add this variable?

Reply:
Not needed now, it's an artifact of an earlier revision. Removed.